src/w1r3/src/main.rs

```diff
@@ -702,6 +704,10 @@ struct Args {
     #[arg(long, default_value_t = 3)]
     read_count: i32,
 
+    /// Skip the deletes.
+    #[arg(long)]
+    no_delete: bool,
```

Comment:
consider calling this `delete` with a default value of `true`.

Reply:
I could not figure out how to get `--delete=false` to work. When we do, we can change it.